experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/VariantManagerImpl.kt

```diff
@@ -106,12 +106,20 @@ class VariantManagerImpl @Inject constructor(
         if (entity.key == "sc" || entity.key == "se") {
             return { isSerpRegionToggleCountry() }
         }
-        if (entity.localeFilter.isEmpty()) {
-            return { noFilter() }
+
+        var matchesLocaleFilter = true
+        var matchesAndroidVersionLocale = true
```

Comment:
This name is confusing, the filter is only for the version, not for the locale.

Reply:
oops! I'll remove the Locale part